content/articles/how-to-build-a-task-manager-with-angular/index.md

```diff
@@ -0,0 +1,473 @@
+
+- You should have [Node.js](https://nodejs.org/en/) installed on your computer.
+- You should be fairly familiar with [Angular](https://angular.io/docs) and JavaScript. Knowledge of [TypeScript](https://www.typescriptlang.org/) is useful but not requisite.
+
+### Key takeaways
+At the end of this tutorial, you should be able to:
+-  create and structure an angular project.
+-  create a component in Angular.
+-  build a simple application.
+-  save items to local storage
+
+
+### What is a Task Manager?
+A task manager writes, organizes, and rearranges tasks more efficiently. The task manager we will build will allow the user to add a task, mark the task as complete, and delete the task.
+
+
+### Create a new angular project
+Before creating a new angular project, one will need to install angular CLI. Run `npm install -g @angular/cli` on the terminal or command prompt to install the CLI.
+
+   ```npm install -g @angular/cli
+   ```
+
+After installing the CLI, you can now create a new angular project. Run the CLI command `ng new *project name*` to do this. We will use `todoApp` as the project name for this article.
+
+    ``` ng new todoApp 
+    ```
+ Accept the defaults by pressing the enter key.
+
+The Angular CLI will install the required Angular npm packages and other dependencies for the project.
+
+The Angular CLI has a server to serve and build applications locally.
+To serve your application to the browser, you will first navigate to the workspace folder. Run `cd todoApp` to do so. 
+Next, run the command `ng serve -o` and wait for some minutes; your application will launch in the browser.
+
+
+    ```
+    cd todoApp
+    ng serve -o
+    ```
+
+The `ng serve` command sets up the server, observes the files, and rebuilds the app as you change those files.
+
+
+### Create a component
+Components are the main constituents of an angular application. Each component consists of;
+
+- An HTML template: states what is rendered on the page.
+- A Typescript class: defines the behavior of the template.
+- A CSS file: it is used to style the template.
+
+Components can be created manually or by using the Angular CLI. For this article, it will be created using the latter. You can start by running the command below on your terminal.
+
+    ```
+    ng generate component todo
+    
+    // OR
+    
+    ng g c todo
+    ```
+
+The CLI command above tells the Angular CLI to generate a `todo` component.
+Now in your code, check the `app` folder, which is in the `src` folder, and you will notice a `todo` folder has been created. This `todo` folder contains the HTML, CSS, and TypeScript files, as shown below.
+
+![Angular Setup](/how-to-build-a-task-manager-with-angular/angular_setup.png)
+
+
+After generating the component, it will be automatically imported into the app module. The component is also added to the `declarations` array. You should check the `app.module.ts` file to see this.
+
```

Comment:
This is not an intro to Angular tutorial, and we assume the reader already have basics in Angular!